timeboost/auctioneer.go

```diff
@@ -228,6 +232,15 @@ func NewAuctioneerServer(ctx context.Context, configFetcher AuctioneerServerConf
 	if err = roundTimingInfo.ValidateResolutionWaitTime(cfg.AuctionResolutionWaitTime); err != nil {
 		return nil, err
 	}
+	
+	bufferSize := cfg.BidsReceiverBufferSize
+	if bufferSize == 0 {
```

Comment:
1. Out of curiosity - why did you abandon returning an error?
2. Please add a note to a help string in `AuctioneerServerConfigAddOptions` that if someone passes `0`, it will fall back to the default 100k